### PR TITLE
Stat tracker

### PR DESCRIPTION
**Pull request by GregJMitchell**
This Branch:
- Reads from game, team, and game_team csv files
- Creates game, team, and game_team class files
- Creates stat tracker class

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Game
```
Comment (s-steel) [inline lib/game.rb:1]: Initializing game class

File: lib/game_teams.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+class GameTeams
```
Comment (s-steel) [inline lib/game_teams.rb:1]: Initialize game_team class

File: lib/team.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class Team
```
Comment (s-steel) [inline lib/team.rb:1]: Initialize team class

File: test/stat_tracker_test.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      game_teams: game_teams_path
+    }
+
+    stat_tracker = StatTracker.from_csv(locations)
+    require "pry"; binding.pry
+
+    assert_equal 7441, stat_tracker.games.length
+    assert_equal 32, stat_tracker.teams.length
+    assert_equal 14882, stat_tracker.game_teams.length
+  end
```
Comment (s-steel) [inline test/stat_tracker_test.rb:24]: I'd like to explore adding more tests here to make sure that all the data is properly pulled in.  I'll work on this on a new branch.

**Review by s-steel — APPROVED** [review]
I'd like to explore adding more tests within the stat_tracker_test to make sure that all the data is properly pulled in.  I'll work on this on a new branch.

File: lib/game_teams.rb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+              :powerPlayOpportunities,
+              :powerPlayGoals,
+              :faceOffWinPercentage,
+              :giveaways,
+              :takeaways
+  def initialize(row)
+    row.each do |k, v|
+      instance_variable_set("@#{k}" , v)
+    end
+    @faceOffWinPercentage = @faceOffWinPercentage.to_f
```
Comment (c-bartell) [inline lib/game_teams.rb:21]: Should we go back later and refactor the instance variables from pascalCase to snake_case?

**Review by c-bartell — COMMENTED** [review]
It has my vote for merger.  We can worry about the pascalCase stuff in refactor.

**Pull request merged**